tutorial/markdown/java/spring-boot.md

```diff
@@ -73,152 +122,261 @@ Once the site is up and running you can launch your browser and go to the Swagge
 
 ## What We'll Cover
 
-A simple REST API using Spring Boot and the Couchbase SDK version 3.x with the following endpoints:
+A simple REST API using Spring Boot and the `Couchbase SDK version 3.x` with the following endpoints:
 
-- [POST a Profile](#post-a-profile) – Create a new user profile
-- [GET a Profile by Key](#get-a-profile-by-key) – Get a specific profile
-- [PUT Profile](#put-profile) – Update a profile
-- [DELETE Profile](#delete-profile) – Delete a profile
-- [GET Profiles by Searching](#get-profiles-by-searching)  – Get all profiles matching first or last Name
+- Create new airlines with essential information.
```

Comment:
Can you link to the relevant parts of the doc?

Reply:
Added hyperlinks to the parts of the doc